docs/source/configuration/overview.mdx

```diff
@@ -1115,15 +1115,15 @@ See [Demand Control](../executing-operations/demand-control) to learn how to ana
 
 ### Early cancel
 
-Up until [Apollo Router 1.43.1](https://github.com/apollographql/router/releases/tag/v1.43.1), when the client closed the connection without waiting for the response, the entire request was cancelled and did not go through the entire pipeline. Since this causes issues with request monitoring, the Router introduced a new behaviour in 1.43.1. Now, the entire pipeline is executed if the request is detected as cancelled, but subgraph requests are not actually done. The response will be reported with the `499` status code, but not actually sent to the client.
+Up until [Apollo Router Core v1.43.1](https://github.com/apollographql/router/releases/tag/v1.43.1), when the client closed the connection without waiting for the response, the entire request was cancelled and did not go through the entire pipeline. Since this causes issues with request monitoring, the router introduced a new behavior in 1.43.1. Now, the entire pipeline is executed if the request is detected as cancelled, but subgraph requests are not actually done. The response will be reported with the `499` status code, but not actually sent to the client.
```

Comment:
Another confusing occurrence for me - Core or GraphOS? If we use GraphOS when applicable to both, this should be GraphOS

Reply:
When applicable to both, use the generic lower-case "router".